client/my-sites/site-settings/site-owner-transfer/use-administrators.ts

```diff
@@ -0,0 +1,42 @@
+import useUsersQuery from 'calypso/data/users/use-users-query';
+
+interface User {
+	ID: number;
+	login: string;
+	email: string;
+	linked_user_ID: number;
+}
+
+interface Page {
+	found: number;
+	users: User[];
+}
+
+interface UserResponse {
+	pages: Page[];
+}
+
+interface UseAdministratorsParams {
+	siteId: number;
+	excludeUserIDs?: number[];
+}
+
+export function useAdministrators( { siteId, excludeUserIDs }: UseAdministratorsParams ) {
+	const queryResult = useUsersQuery(
+		siteId,
+		{ role: 'administrator' },
+		{
+			select: ( data: UserResponse ) => {
+				const users = data?.pages?.[ 0 ]?.users || [];
+				if ( excludeUserIDs ) {
+					return users.filter( ( user ) => ! excludeUserIDs.includes( user.linked_user_ID ) );
```

Comment:
@sejas , I noticed something strange.
My user is shown in the dropdown, however, he shouldn't. Changing `linked_user_ID` to `ID` solves the issue for me, locally  

```suggestion
					return users.filter( ( user ) => ! excludeUserIDs.includes( user.ID ) );
```

Reply:
I was using `linked_user_ID` because that's the value for atomic sites, but it doesn't work for simple.
So I changed the filtering method to use the email: https://github.com/Automattic/wp-calypso/pull/78309/commits/44addb76a73d44adb0c22efbc056d39e41c0bdd4

Reply:
Great! I see there is a corner case in slack, however, I will approve it. Thanks for taking care of it!